src/server/common/model/answer/text-area/text-area.js

```diff
@@ -10,19 +10,20 @@ import { NotImplementedError } from '../../../helpers/not-implemented-error.js'
  * @returns {Joi.Schema}
  */
 const textAreaSchema = ({ payloadKey, validation }) => {
-  let stringValidation = Joi.string().trim().max(validation.maxLength.value)
+  let stringValidation = Joi.string()
+    .required()
+    .trim()
+    .max(validation.maxLength.value)
```

Comment:
Would be preferable to be a separate PR, but this looks like a really good change :+1: